poi-ooxml/src/main/java/org/apache/poi/xdgf/usermodel/shape/ShapeDebuggerRenderer.java

```diff
@@ -40,9 +39,9 @@ public void setDebugAcceptor(ShapeVisitorAcceptor acceptor) {
     }
 
     @Override
-    protected Path2D drawPath(XDGFShape shape) {
```

Comment:
does this compile? Regardless, I don't think we should change protected method signatures in a patch release. Could you consider adding new private methods and leave the protected methods alone.

Reply:
Hey @pjfanning! Good catch. I completely forgot that `ShapeRenderer` class might be used outside of `poi` codebase itself. 

Can you please check out the last revision? I reverted changing the method signature and preserved its behaviour. Now, it draws all the sections but returns only first drawn path.